packages/components/inputs/rich-text-input/src/rich-text-input.visualspec.js

```diff
@@ -375,5 +375,5 @@ describe('RichTextInput', () => {
 
     await input.type('Okay dokey');
     await wait(() => getByText(doc, 'Okay dokey'));
-  });
+  }, 20000);
```

Comment:
This seems to help with the visual tests for the rich text editor (hopefully)

Reply:
The default timeout of rtl is quite low. Maybe globally increase it as in app-kit and then the MC. They changed it at some point to a lower value.

Reply:
Yeah thanks. The rich text input seems to be kind of flaky recently (for example: https://github.com/commercetools/ui-kit/runs/986022468?check_suite_focus=true), I think we need to find a way to make it more stable first.
Increasing the timeout was just an attempt.